spring/camel-with-narayana-spring-boot/pom.xml

```diff
@@ -90,6 +90,20 @@
             <artifactId>camel-jpa-starter</artifactId>
             <version>${camel.version}</version>
         </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-dbcp2</artifactId>
+            <exclusions>
```

Comment:
by interest: why is needed to exclude this artifact here?

Reply:
It just copy from https://github.com/spring-projects/spring-boot/blob/master/spring-boot-project/spring-boot-dependencies/pom.xml#L1330. I think it could be excluded to resolve the conflict artifacts.